FW/Presentations/Lab_Guides/_L_C_01_Build_Setup_Download_EDK_II_Linux_LabGuide.md

```diff
@@ -0,0 +1,328 @@
+
+Example Using Clear Linux* Project 
+
+The following need to be accessible for building Edk2, From the terminal prompt (Ctrl-Alt-T):
+```bash
+$ sudo swupd bundle-add devpkg-util-linux
+```
+
+Devpkg-util-linux - includes bundles for developer tools for writing  "C" Applications included: gcc, nasm, uuid, etc.
+
+```bash
+$ sudo swupd bundle-add kvm-host
+```
+
+Qemu - Emulation with Intel architecture with UEFI Shell 
+
+---
+## Slide 6 Setup Simics Environment
+
+Download and Install the Simics Simulator (both Package Manager & Simics-6 Packages) [Simics® Simulator Public Release](https://www.intel.com/content/www/us/en/developer/articles/tool/simics-simulator.html)
+
+Setup the Simics Simulator
+
+[Simics® Simulator Installation and Get Started](https://www.intel.com/content/www/us/en/developer/articles/guide/simics-simulator-installation.html)
+
+Open the "Intel Simics Package Manager"
+
+1. Using the Start menu in Windows, Left Click on "Windows Key" Lower Left
+2. Scroll down from the scroll bar on the right until "Intel Simics Package Manager"
+
+Here is a snapshot of Intel Simics Package Manager with "My Simics Project 1" created
+
+
+---
+## Slide 7 Create QEMU Run Script
+
+1. Create a run-ovmf directory under the home directory
+
+```bash
+$ cd ~
+$ mkdir ~run-ovmf
+$ cd run-ovmf
+```
+
+2. Create a directory to use as a hard disk image
+```bash
+$ mkdir hda-contents
+```
+
+3. Create a Linux shell script to run the QEMU from the run-ovmf directory
+```bash
+$ gedit RunQemu.sh
+```
+
+Type in the following for the file RunQemu.sh
+```
+qemu-system-x86_64 -pflash bios.bin -hda fat:rw:hda-contents -net none -debugcon file:debug.log -global isa-debugcon.iobase=0x402 
+```
+
+4. Save and Exit
+
+
+---
+## Slide 8 Download EDK II Source
+### Download Source and Lab Materials
+
+<br>
+
+---
+## Slide 9 Create Workspace Directory
+
+Open Windows Command Prompt for VS
+Make new directory for Workspace:
+
+```bash
+$ cd /
+$ mkdir fw
+$ cd fw
+$ mkdir edk2-ws
+$ cd edk2-ws
+```
+
+---
+## Slide 10 Download EDK II Open Source
+### Download the open source EDK II from GitHub
+
+Note:  if behind a firewall, set PROXYS FIRST (example shows for Intel Corp. -- Maybe different for your Corp. or GEO)
+
+```bash
+$ git config --global https.proxy proxy-dmz.intel.com:912
+$ git config --global http.proxy proxy-dmz.intel.com:911
+```
+
+From the command prompt use "git clone" to download following repos
+\#Edk2 - main core code
+
+```bash
+$ git clone -b Edk2Lab_22Q3 https://github.com/tianocore-training/edk2.git
+```
+\#EDK II "C" Library Repo
+
+```bash
+$ git clone https://github.com/tianocore/edk2-libc.git
+```
+\#EDK II Platforms Repo
+
+```bash
+$ git clone https:://github.com/tianocore/edk2-platforms.git
+```
+\#EDK II Non-OSI (Stand alone Binaries)
+
+```bash
+$ git clone https:://github.com/tianocore/edk2-non-osi.git
+```
+\#Intel FSP
+```bash
+$ git clone https://github.com/IntelFsp/FSP.git
+```
```

Comment:
Change to 
$ git clone https://github.com/Intel/FSP.git

Issue was in slide deck too